frontend/src/components/common/Footer.tsx

```diff
@@ -7,12 +7,11 @@ const Footer = (): JSX.Element => (
   <>
     <Divider pt={{ base: "79px", md: "117px" }} />
     <Container
-      maxWidth={{ base: "default", md: "70%", lg: "80%" }}
+      maxWidth={{ lg: "80%" }}
```

Comment:
the footer seems to still be misaligned for me: 
my guess is probably to change the `justifyContent="space-evenly"` to `space-between` and then adjust the width back? 
![image](https://user-images.githubusercontent.com/19617248/151733625-6a2573b7-1f08-459c-8042-9413eaed7872.png)

Reply:
or instead of adjusting width back, pass it the same variant as the header?

Reply:
made a fix! could you double check to see if it's what you want?

Reply:
looks great! 😄